### PR TITLE
issue 1640 : context -> map / carte / mapa

### PR DESCRIPTION
**Pull request by MaelREBOUX**

### CODE REVIEW

**Comment by MaelREBOUX** [comment]
see #1640

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js [hunk 1]
```diff
@@ -455,33 +455,33 @@ OpenLayers.Lang.de = OpenLayers.Util.extend(OpenLayers.Lang.de, {
...
     "Access count:": "Anzahl Zugriffe:",
     "Permalink:": "Permalink:",
-    "My contexts": "Meine Kontexte",
+    "My contexts": "Meine karten",
     "Created": "Erstellung",
     "Accessed": "Zugriff",
     "Count": "Zugriffe",
     "View": "Anzeigen",
-    "View the selected context": "Ausgewählten Kontext anzeigen",
+    "View the selected context": "Ausgewählten karte anzeigen",
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js]: karte -> Karte ?

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js [hunk 2]
```diff
@@ -455,33 +455,33 @@ OpenLayers.Lang.de = OpenLayers.Util.extend(OpenLayers.Lang.de, {
...
     "Download": "Laden",
-    "Download the selected context": "Ausgewählten Kontext herunterladen",
+    "Download the selected context": "Ausgewählten karte herunterladen",
     "Delete": "Löschen",
-    "Delete the selected context": "Ausgewählten Kontext löschen",
-    "Failed to delete context": "Löschen des Kontexts fehlgeschlagen",
-    "Manage my contexts": "Meine Kontexte verwalten",
+    "Delete the selected context": "Ausgewählten karte löschen",
+    "Failed to delete context": "Löschen des karten fehlgeschlagen",
+    "Manage my contexts": "Meine karten verwalten",
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js]: Hier auch ...

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/es.js [hunk 3]
```diff
@@ -467,34 +467,33 @@ OpenLayers.Lang.es = OpenLayers.Util.extend(OpenLayers.Lang.es, {
...
+    "View the selected context": "Visualizar el mapa seleccionada (ojo : remplazará el mapa actuale)",
     "Download": "Descargar",
-    "Download the selected context": "Descargar el contexto seleccionado",
+    "Download the selected context": "Descargar el mapa seleccionada",
     "Delete": "Borrar",
-    "Delete the selected context": "Borrar el contexto seleccionado",
-    "Failed to delete context": "El contexto no pudo ser borrado",
-    "Manage my contexts": "Manejar mis contextos",
+    "Delete the selected context": "Borrar el mapa seleccionada",
+    "Failed to delete context": "el mapa no pudo ser borrado",
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/es.js]: Missing uppercased first letter

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js (hunk 1 above)
Comment (MaelREBOUX) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js]: really don't know / can't remember if Maj is mandatory for common names in german

Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js]: it is.

Comment (MaelREBOUX) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js]: done

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/es.js (hunk 3 above)
Comment (MaelREBOUX) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/es.js]: done

**Comment by fvanderbiest** [comment]
perfect !

**Pull request merged**